pkg/cluster/px/px.go

```diff
@@ -1074,3 +1075,19 @@ func parseMajorMinorVersion(version string) (string, error) {
 
 	return matches[1], nil
 }
+
+// isPXOCIImage checks if given image name is an oci monitor image
+func isPXOCIImage(pxImageName string) bool {
+	matched, _ := regexp.MatchString(".*registry.connect.redhat.com/portworx/px-enterprise.+|.+oci-monitor.+", pxImageName)
```

Comment:
This works, but ... regexp objects are normally very expensive to create (but reusable).

Recommend setting them up as global variables (so they init only once) and then using them via global var (see https://golang.org/pkg/regexp/#Regexp.MatchString)